packages/elements/src/chart/index.ts

```diff
@@ -10,19 +10,30 @@ import { customElement } from '@refinitiv-ui/core/decorators/custom-element.js';
 import { property } from '@refinitiv-ui/core/decorators/property.js';
 import { query } from '@refinitiv-ui/core/decorators/query.js';
 import { VERSION } from '../version.js';
+import { color as parseColor } from '@refinitiv-ui/utils/color.js';
+
+type DatasetColors = {
+  solid: string | string[];
+  opaque: string | string[];
+};
 
 import {
   merge,
   MergeObject
 } from './helpers/index.js';
 
 import {
-  Chart as ChartJS
+  Chart as ChartJS,
+  Plugin,
+  ChartType,
```

Comment:
`ChartType` is unused.

Reply:
It currently uses to let chart know the plugin. If we remove this, Typescript will say error that chartjs.options doesn't know the plugin.
```
declare module 'chart.js' {
  interface PluginOptionsByType<TType extends ChartType> {
    efchart?: {
      enable: boolean
    }
  }
}
```